plugins/test/unit/plugins/importers/test_web.py

```diff
@@ -48,14 +48,29 @@ def test_import(self):
             Mock()
         ]
         conduit = Mock()
+
+        # test
+        importer = WebImporter()
+        report = importer.import_units(None, None, conduit, None, units)
+
+        # validation
+        self.assertEqual(conduit.associate_unit.call_args_list, [((u,), {}) for u in units])
+        self.assertEqual(report, units)
+
+    def test_import_all(self):
+        units = [
+            Mock(),
+            Mock(),
+            Mock()
+        ]
+        conduit = Mock()
         conduit.get_source_units.return_value = units
 
         # test
         importer = WebImporter()
         report = importer.import_units(None, None, conduit, None)
 
         # validation
-        conduit.get_source_units.assert_called_once_with()
```

Comment:
Why this assertion is not needed any more? 
It looks like `conduit.get_source_units` should be called when `units` are `None`.
Or it is tested implicitly by the next assertion where unit association is called with proper arguments?